README.md

```diff
@@ -1,2 +1,56 @@
-# bubble-sorting
-This is a Ruby code that allows you to sort an array using the bubble sorting method.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+## Ruby Bubble-Sort and  Bubble-Sort_by project
+
+> This is a project made with ruby to order items inside of a array, one order items with numbers and the other one order strings inside of the array using a yield feature.
+
```

Comment:
Your README appear to have few defects, you can use [Grammarly](https://www.grammarly.com) to crosscheck and improve your grammar.